src/main/java/nl/hsac/fitnesse/fixture/fit/ReadFileFromFtpServerFixture.java

```diff
@@ -15,43 +17,60 @@
     private static final String PASSWORD_KEY = "password";
     private static final String FILE_PATH_KEY = "file";
     private static final String LINES_COUNT_KEY = "nrOfLines";
+    protected static final String IS_SFTP = "sftp";
```

Comment:
Can you add usage of this new column to 'HsacExamples.FitTests.ReadFileFromFtpServerFixture'? And maybe also some documentation on what needs to be configured to make sftp work (I expect stuff like adding keys to keystores)?
Is there a reason this one is protected while others are private (only for test?)

Reply:
I will update the example. This implementation only supports username/password so no adding of keys to keystores.
The value is now protected so it can be used in the test and i don't think it can do harm if subclasses can use this value.

Reply:
Ok